guides/using-iron-icons.md

```diff
@@ -0,0 +1,243 @@
+[//]: # (instead of trying to display inside of document, create working examples with full index.html)
+[//]: # (how to style icons embedded in other elements?)
+[//]: # (delete video if it mentions iron-iconset*)
+[//]: # (color not inheriting on paper-icon-button)
+
+
+<link rel="import" href="/bower_components/google-youtube/google-youtube.html">
+
+## Introduction
+
+In this guide we will teach you how to display icons and how to use the 
+Polymer team's collection of icons in your project.
+
+We discuss two elements:
+
+* `iron-icon`, for displaying and styling an icon
+* `iron-icons`, for using the Polymer team's collection of icons
+  in your project
+
+## Installation
+
+Install the icon elements with Bower. We'll assume that your project can access 
+these elements from `/bower_components/`.
+
+```bash
+bower install PolymerElements/iron-icon 
+bower install PolymerElements/iron-icons 
+```
+
+## Displaying an icon with `iron-icon`
+
+To display an icon, import `iron-icon` and specify the source image using
+either the `src` attribute if the image is not part of an icon set,
+or the `icon` attribute if it is.
+
+    ...
+    <link rel="import" href="/bower_components/iron-icon/iron-icon.html">
+    ...
+    <iron-icon src="/images/polymer.svg"></iron-icon>
+    ...
+
+Produces: 
+
+<iron-icon src="/images/polymer.svg"></iron-icon>
+
+The source image is set as the icon's background and is scaled to fit the icon 
+size. It can be bitmap or SVG. `iron-icon` expects the source image to be square.
+The default icon size is 24 pixels by 24 pixels. Use CSS to set the icon size.
+
+    ...
+    <iron-icon src="/images/polymer.svg" style="width: 24px; height: 24px;"></iron-icon>
+    <iron-icon src="/images/polymer.svg" style="width: 32px; height: 32px;"></iron-icon>
+    <iron-icon src="/images/polymer.svg" style="width: 48px; height: 48px;"></iron-icon>
+    ...
+
+Produces: 
+
+<iron-icon src="/images/polymer.svg" style="width: 24px; height: 24px;"></iron-icon>
+<iron-icon src="/images/polymer.svg" style="width: 32px; height: 32px;"></iron-icon>
+<iron-icon src="/images/polymer.svg" style="width: 48px; height: 48px;"></iron-icon>
+
+### Globally styling width and height
+
+To create style rules that affect the size of all icons, create a theme file
+and use the `--iron-icon-width` and `iron-icon-height` mixins.
+
+```html
+<!-- custom-icon.html -->
+<style is="custom-icon">
+  iron-icon {
+    --iron-icon-width: 100px;
+    --iron-icon-height: 100px;
+  }
+</style>
+```
+
+Import the theme file into your page and style the icons with your custom class:
+
+```html
+   ...
+   <link rel="import" href="custom-icon.html">
+   ...
+   <iron-icon class="custom-icon" icon="accessibility"></iron-icon>
+```
+
+## Using the Polymer team's icon collection, `iron-icons`
+
+The Polymer team has created a large collection of free, SVG icons that 
+you can use in your own project. These icons are distributed as an element
```

Comment:
element => package (or "component" would also work)

Terminology is important here, because in this case we're talking about both a bower package (iron-icons) which contains several icon sets you can import, including iron-icons.html, social-icons.html, etc. None of these are actually custom elements (you can call them files, or imports, or icon sets).